src/main/java/leduc/command/TodoCommand.java

```diff
@@ -33,26 +35,40 @@ public  TodoCommand(String user){
      * @param storage leduc.storage.Storage which deals with loading tasks from the file and saving tasks in the file.
      * @throws EmptyTodoException Exception caught when the description of the todo list is not given by the user.
      * @throws FileException Exception caught when the file can't be open or read or modify
+     * @throws PrioritizeLimitException Exception caught when the new priority is greater than 9 or less than 0.
      */
-    public void execute(TaskList tasks, Ui ui, Storage storage) throws EmptyTodoException, FileException {
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws EmptyTodoException, FileException, PrioritizeLimitException {
         String userSubstring;
         if(callByShortcut){
             userSubstring = user.substring(TodoCommand.todoShortcut.length());
         }
         else {
             userSubstring = user.substring(4);
         }
-        if (userSubstring.isBlank()) {
-            throw new EmptyTodoException();
+        String[] prioritySplit = userSubstring.split("prio");
+        TodoTask newTask = null;
+        int priority = -1 ;
+        if (prioritySplit.length != 1 && prioritySplit[1].trim().matches("\\d+")){
+            priority = Integer.parseInt(prioritySplit[1].trim());
+            if (priority < 0 || priority > 9) {
+                throw new PrioritizeLimitException();
+            }
+            if (prioritySplit[0].isBlank()){
+                throw new EmptyTodoException();
+            }
+            newTask = new TodoTask(prioritySplit[0],"[✗]",priority);
         }
-        else {
-            TodoTask newTask = new TodoTask(user.substring(TodoCommand.todoShortcut.length()).trim());
-            tasks.add(newTask);
-            storage.save(tasks.getList());
-            ui.display("\t Got it. I've added this task:\n\t   "
-                    + newTask.toString() +
-                    "\n\t Now you have " + tasks.size() + " tasks in the list.");
+        else { // The description of the todo task could contain "prio"
```

Comment:
if there is no int after the prio you throw EmptyTodoException whereas before for deadlineCommand and EventCommand, you throw a PrioritizeLimitException

Reply:
No, if it is only todo prio, without int after, I throw anything, the todo task with the description prio and priority 5 will be created, I already test it in Junit test

Reply:
ok I understand, but not clear